custom_components/mqtt_basic_sync/__init__.py

```diff
@@ -27,9 +27,19 @@
 DEFAULT_TOPIC = 'home-assistant/mqtt_example'
 
 # Schema to validate the configured MQTT topic
-CONFIG_SCHEMA = vol.Schema({
-    vol.Optional(CONF_TOPIC, default=DEFAULT_TOPIC): mqtt.valid_subscribe_topic
-})
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(
```

Comment:
I'd keep it optional since there's a default.

Reply:
```suggestion
                vol.Optional(
```